scripts/build_db_image

```diff
@@ -5,17 +5,15 @@
 
 dockerfile="$1"
 repo_name="$2"
-tag="${3:-$CIRCLE_SHA1}"
+ecr_backend="${AWS_ACCOUNT_ID}.dkr.ecr.us-west-2.amazonaws.com/$repo_name"
+tag="${ecr_backend}:${CIRCLE_SHA1}"
```

Comment:
"latest" is still being used for the db_clean image: https://github.com/CMSgov/easi-app/blob/8b69a6f009dfbac25d49cb0af68910910cb0d055/.circleci/config.yml#L113
so the changes in this script probably need to be tweaked a bit.